lib/galaxy/selenium/navigates_galaxy.py

```diff
@@ -1696,6 +1702,7 @@ def run_tour(self, path, skip_steps=None, sleep_on_steps=None, tour_callback=Non
             tour_callback = NullTourCallback()
 
         self.home()
+        self.use_legacy_history()
```

Comment:
That means we're not testing tours, I suppose that should at least be a todo item ?